sagemaker/launch/launch_realtime_endpoint.py

```diff
@@ -0,0 +1,313 @@
+        Amazon ECR in the same region specified in the `region` argument.
+    role: str
+        The ARN string of an AWS account ARN that has SageMaker execution and model registry full
+        access role.
+    instance_type: str
+        The string type of a SageMaker instance type. The default value is \"ml.c6i.xlarge\".
+    instance_count: int
+        The number of SageMaker instances to be deployed for the endpoint.
+    restore_model_path: str
+        The local folder path where GraphStorm trained model artifacts were stored, e.g.,
+        \"save_model/epoch-1/\".
+    model_yaml_config_file: str
+        The YAML file path. This YAML file is the one that was stored by GraphStorm model training
+        pipeline during model training. It is NOT the one used as one of the arguments
+        for GraphStorm model training.
+    graph_json_config_file: str
+        The JSON file path. This JSON file is the one that was stored by GraphStorm graph
+        building tools, e.g., gconstruct and GSProcessing. It is NOT the one used as one of the
+        arguments for GraphStorm graph building.
+    upload_tarfile_s3: str
+        The S3 location to upload the packed and compressed model tar file. This location should
+        be in the same region specified in the `region` argument.
+    infer_task_type: str
+        The name of real time inference task. Options include \"node_classification\".
+    model_name: str
+        A string to define the name of a SageMaker inference model. This name string must follow
+        SageMaker's naming format, which is ^[a-zA-Z0-9]([\-a-zA-Z0-9]*[a-zA-Z0-9])$. The default
+        value is \"GSF-Model4Realtime\".
+    model_tarfile_s3: str
+        The S3 location where a custom model tar file is stored. If provided, this script will
+        use this model tarfile to deploy a SageMaker endpoint. This S3 location should be in the
+        same region specified in the `region` argument.
+    entrypoint_file_name: str
+        The name of a Python file. SageMaker will use this file as the entry point of an endpoint.
+        This is name is required if the `model_tarfile_s3` is provided. If not, this script will
+        GraphStorm's default entry point file according to the `infer_task_type`.
+    
+    """
+    # ================= prepare model artifacts ================= #
+    # prepare sessions
+    boto_session = boto3.Session(region_name=input_args.region)
+    sm_session = sm.Session(boto_session=b3_session)
+
+    # If users provide the S3 tar file path already, directly use it for model creation
+    if input_args.model_tarfile_s3:
+        entrypoint_file_name = input_args.entrypoint_file_name
+
+        model_url_s3 = input_args.model_tarfile_s3
+        assert check_tarfile_s3_object(args.model_tarfile_s3) is True, \
+            (f'Not find a tar file in the given S3 URL: {args.model_tarfile_s3}...')
```

Comment:
```suggestion
        assert check_tarfile_s3_object(args.model_tarfile_s3), \
            (f'Did not find a tar file in the given S3 URL: {args.model_tarfile_s3}...')
```

Reply:
Will not do this check, but let users take care of it.